python/ml4ir/base/features/preprocessing.py

```diff
@@ -130,6 +133,8 @@ def preprocess_text(
     Output:
         >>> "abcabc123"
     """
+    if remove_quotes:
```

Comment:
This is another minor change I strangely had in my git stash. Can move to another PR if needed.